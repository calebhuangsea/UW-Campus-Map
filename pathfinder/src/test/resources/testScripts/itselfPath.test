# find path between a node and itself

CreateGraph graph1

AddNode graph1 n1

AddEdge graph1 n1 n1 1.1111

FindPath graph1 n1 n1

