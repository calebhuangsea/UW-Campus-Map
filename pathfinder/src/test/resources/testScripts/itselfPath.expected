# find path between a node and itself

created graph graph1

added node n1 to graph1

added edge 1.111 from n1 to n1 in graph1

path from n1 to n1:
total cost: 0.000

